architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,540 @@
+[[architecture-additional-concepts-dynamic-admission-controllers]]
+= Dynamic Admission Controllers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Overview
+
+Dynamic admission controllers allow you to deploy custom (out-of-tree)
+xref:../../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control] plug-ins
+and configure them at runtime. 
+
+Admission control plug-ins intercept requests to the master API prior to persistence of a resource, but after the request is authenticated and authorized. The admission stage of the API server processing is one of the most powerful tools for securing an {product-title} cluster by restricting the objects that can be created. However, creating custom
+admission controllers cannot be easily added to {product-title} for the following reasons:
+
+* Standard plug-in admission controllers requires building a custom version of {product-title}.
+* They are only configurable when the API server starts up.
+
+Dynamic admission controllers are designed to work around these limitations to allow you to easily use your own admission controllers with {product-title}.  
+ 
+You can create admission controllers that use xref:architecture-additional-concepts-dynamic-admission-webhooks[external webhooks] to make admission decisions. 
```

Comment:
now we have: controllers, plugin, standard plugins (what are non-standard plugins?), dynamic controllers, webhooks.